README.md

```diff
@@ -23,7 +23,7 @@ Prerequisites:
 
 Then:
 
-1. Download the `.zip` file from the latest [release](releases/latest) and extract it to:
+1. Download the `.zip` file from the latest[release](https://github.com/hlaueriksson/Community.PowerToys.Run.Plugin.Install/releases) and extract it to:
```

Comment:
Can you keep the space between `latest` and `[release]`?

And keep the slug `/latest` in the URL?